src/slices/AccountSlice.ts

```diff
@@ -26,7 +34,6 @@ interface IUserBalances {
     fsohm: string;
     wsohm: string;
     fiatDaowsohm: string;
-    wsohmAsSohm: string;
```

Comment:
wsohmAsSohm was removed throughout this file, so despite the conflicts, I removed it as well.

Reply:
I guess that seems fine? We didn't use it anywhere so if they removed it I'm hoping they replaced it with something else where they need it and it shouldn't affect Tyche?